replica/table.cc

```diff
@@ -2004,6 +2007,9 @@ void table::update_effective_replication_map(locator::effective_replication_map_
                           _schema->ks_name(), _schema->cf_name(), old_tablet_count, new_tablet_count);
             handle_tablet_split_completion(old_tablet_count, _erm->get_token_metadata().tablets().get_tablet_map(table_id));
         }
+
+        auto* tablet_sg_manager = dynamic_cast<tablet_storage_group_manager*>(_sg_manager.get());
+        tablet_sg_manager->update_effective_replication_map(_erm);
     }
```

Comment:
Isn't handle_tablet_split_completion very similar? Should the functionality be merged?

Reply:
Yes, it can be done along with (or after) "table: move compaction_group_list and storage_group_vector to storage_group_manager"

Reply:
done in next version